src/install.rst

```diff
@@ -391,129 +400,20 @@ Finally, try running Augur and Auspice.
-      Make sure there are no errors or warnings reported.
-
-      You can forcibly setup the Conda runtime again by running:
-
-      .. code-block:: bash
-
-         nextstrain setup --force conda
-
-      This should rarely be necessary, but may help if you find yourself with a broken runtime.
-
-
-   .. group-tab:: Ambient
 
-      Running ``nextstrain check-setup ambient`` will report potential issues.
-      Make sure there are no errors or warnings reported.
-
-      Ensure that you've activated your custom Conda environment with ``conda activate nextstrain``.
-
-      If Conda fails to install or update Nextstrain using the commands in the other sections above, it's possible that Conda itself is out-of-date or that Conda cannot figure out how to resolve the environment's dependencies.
-      Starting from scratch often fixes problems with Conda environments.
-      To start over with a new Nextstrain environment, delete your current environment.
-
-      .. code-block:: bash
-
-          conda activate base
-          conda env remove -n nextstrain
+Next steps
+==========
 
-      Then, repeat the installation instructions above, starting with the update of Conda itself.
+With Nextstrain installed, try :doc:`tutorials/running-a-workflow` next.
 
-If you the above isn't sufficient and you need more help troubleshooting, please post to our `discussion forum <https://discussion.nextstrain.org/c/help-and-getting-started/6>`__ where members of the community and the Nextstrain team can help out.
 
 Alternate installation methods
 ==============================
 
 If you want to :doc:`contribute to the development of Nextstrain </guides/contribute/index>` or if you prefer to manage your own custom environment (e.g., a Conda environment, Docker image, environment modules on a cluster, etc.), see the individual installation documentation for :doc:`Nextstrain CLI <cli:installation>`, :doc:`Augur <augur:installation/installation>`, and :doc:`Auspice <auspice:introduction/install>`.
 
-Uninstall
-=========
-
-We do not have an automated uninstall process currently.
-Instead, follow these manual steps:
 
-   1. If the directory :file:`~/.nextstrain` exists, remove it.
-   2. If using the Docker runtime, remove all ``nextstrain/…`` Docker images::
+Managing an existing installation
+=================================
 
-         docker image rm $(docker image ls -q "nextstrain/*")
-
-      Optionally, uninstall Docker if only used for Nextstrain.
-   3. If using the ambient runtime, remove the ``nextstrain`` custom Conda environment::
-
-         conda env remove -n nextstrain
-
-      Optionally, uninstall Conda if only used for Nextstrain.
-   4. On Windows, optionally, uninstall WSL if only used for Nextstrain.
-
-Next steps
-==========
-
-With Nextstrain installed, try :doc:`tutorials/running-a-workflow` next.
+See :doc:`this page </guides/manage-installation>` for steps to update, troubleshoot, or uninstall Nextstrain tools.
```

Comment:
_non-blocking_

I loathe link text like "this page" or "click here". I think just about anything else is better. Smallest change here might be to just use the page title like we do elsewhere:

```suggestion
See :doc:`/guides/manage-installation` for steps to update, troubleshoot, and uninstall Nextstrain.
```

or we could get a bit more diverged:

```suggestion
See our how-to guide with the :doc:`steps to update, troubleshoot, and uninstall Nextstrain </guides/manage-installation>`.
```

Reply:
Went with the first suggestion in [force-push](https://github.com/nextstrain/docs.nextstrain.org/compare/675e98db2f8e06feb21ae690edad37249fe33d89..427b38c5be7d9649469743731f01a57d3ed9a4b4).